mono/mini/interp/transform.c

```diff
@@ -2129,7 +2132,7 @@ interp_constrained_box (TransformData *td, MonoDomain *domain, MonoClass *constr
 		interp_add_ins (td, MINT_BOX_NULLABLE);
 		td->last_ins->data [0] = get_data_item_index (td, constrained_class);
 		td->last_ins->data [1] = csignature->param_count;
-		td->last_ins->data [2] = (td->sp - 1 - csignature->param_count)->type != STACK_TYPE_MP ? 0 : 1;
+		td->last_ins->data [2] = 1;
```

Comment:
Unrelated ?

Reply:
Loosely related. By switching order of stack I started to get some failures caused by bugs where we were overflowing the stack in a lucky way before. I used the more aggressive asserts to track these down.